app/code/Magento/Checkout/view/frontend/web/template/minicart/content.html

```diff
@@ -27,26 +33,29 @@
                 },
                 click: closeMinicart()
             ">
-        <span translate="'Close'"/>
+        <span data-bind="i18n: 'Close'"/>
     </button>
 
-    <if args="getCartParam('summary_count')">
+    <!-- ko if: getCartParam('summary_count') -->
         <div class="items-total">
-            <span class="count" if="maxItemsToDisplay < getCartLineItemsCount()" text="maxItemsToDisplay"/>
-            <translate args="'of'" if="maxItemsToDisplay < getCartLineItemsCount()"/>
-            <span class="count" text="getCartParam('summary_count')"/>
-                <!-- ko if: (getCartLineItemsCount() === 1) -->
-                    <span translate="'Item in Cart'"/>
-                <!--/ko-->
-                <!-- ko if: (getCartLineItemsCount() > 1) -->
-                    <span translate="'Items in Cart'"/>
-                <!--/ko-->
+            <!-- ko if: maxItemsToDisplay < getCartLineItemsCount() -->
+            <span class="count" data-bind="text: maxItemsToDisplay"/>
+            <!-- /ko -->
+            <!-- ko if: maxItemsToDisplay < getCartLineItemsCount() --><!-- ko i18n: 'of' --><!-- /ko --><!-- /ko -->
+            <span class="count" data-bind="text: getCartParam('summary_count')"/>
+            <!-- ko if: (getCartParam('summary_count') === 1) -->
+                <span data-bind="i18n: 'Item in Cart'"/>
+            <!-- /ko -->
+            <!-- ko if: (getCartParam('summary_count') > 1) -->
+                <span data-bind="i18n: 'Items in Cart'"/>
+            <!-- /ko -->
         </div>
```

Comment:
@hostep i have updated here for correct check items count qty

Reply:
replace getCartLineItemsCount() by getCartParam('summary_count')

Reply:
Ah ok, that wasn't really clear to me, you can always leave a comment saying what you've changed. Sorry about the noise! 🙂

Reply:
Seems like this is a bug fix that should be double checked.

Reply:
@ihor-sviziev i have updated test case for QA